contracts/interfaces/IERC7540.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IERC4626} from "../interfaces/IERC4626.sol";
+
+/**
+ * @dev Interface for the ERC-7540 Asynchronous Tokenized Vaults standard.
+ * https://eips.ethereum.org/EIPS/eip-7540[ERC-7540]
+ */
+interface IERC7540 is IERC4626 {
+    struct Request {
```

Comment:
This is not part of the ERC

Reply:
You can copy the interfaces that match the standard from here: https://github.com/foundry-rs/forge-std/pull/644/files#diff-39b5d73aa19abd1df539c3f24017dd6db331f8ef698e36a3495aa7cbe1ffc89f